daemon-tests/src/lib.rs

```diff
@@ -423,16 +423,22 @@ pub fn dummy_quote() -> Quote {
     }
 }
 
+/// Offer params allowing a single position, either short or long
```

Comment:
Nit: triple-slash is rustdoc and that one has a convention that the first line is a summary description of the documentation: https://brson.github.io/2012/04/14/how-to-rustdoc#the-rules-of-rustdoc

A comment like this I would expect to be double-slash.